src/characters.rs

```diff
@@ -0,0 +1,59 @@
+use crate::AppState;
+use actix_web::{get, web, HttpRequest, HttpResponse, Responder};
+use serde::Serialize;
+use std::collections::HashMap;
+
+#[derive(Debug, Serialize)]
+pub struct Character {
```

Comment:
Instead of renaming the fields used in javascript to match rust, you can add `#[serde(rename_all = "camelCase")]` here to have serde serialize these as camelCase so no changes are required client side.